gap/perm/giant.gi

```diff
@@ -744,7 +744,6 @@ FindHomMethodsPerm.Giant :=
         SetIsRecogInfoForAlmostSimpleGroup(ri,true);
     fi;
     SetNiceGens(ri,StripMemory(res.gens));
-    Setslptonice(ri,res.slpnice);
```

Comment:
Maybe apply the same changes to `gap/matrix/slconstr.gi` where the same pattern occurs, twice -- then afterwards, all occurrences of `slpnice` are eliminated.

Reply:
Nice. Will do!